osconfig_tests/main.go

```diff
@@ -81,6 +80,7 @@ func main() {
 		close(tests)
 	}()
 
+	var testSuites []*junitxml.TestSuite
```

Comment:
Did you mean to populate this?

Reply:
ah.. i had this in a different branch. this was to address a bug. the bug was that the execution of the test_runner never fails even if there is a failure. This is because we first read the testsuites from the channel to create the junit files and the we again try to iterate over the channel to check for failures. but during the second loop, the channel is already empty and so the execution never fails.
I wanted to fix it by dumping the suites to an array and then use it for all iterations.. will send out a different pr for this.